pkg/beeclient/api/api.go

```diff
@@ -185,6 +185,9 @@ func drain(r io.ReadCloser) {
 // responseErrorHandler returns an error based on the HTTP status code or nil if
 // the status code is from 200 to 299.
 func responseErrorHandler(r *http.Response) (err error) {
+	if r.StatusCode == http.StatusAccepted {
+		return ErrRecoveryInitiated
+	}
```

Comment:
Isn't this too specific to a global pinning feature in order to be in used in every request? Status accepted is not a universal equivalence of recovery in the api. I think that this status should be handled specifically only in cases where chunk repair feature exists.

Reply:
you are right. But i could not find any other way to check if global pinning is triggered. I thought you will not get any data when you send 202.. but i get some data... not sure from where... Let me know how this can be handled.